SensorApplication/out/production/SensorApplication/src/SensorPanel.java

```diff
@@ -0,0 +1,36 @@
+import javax.swing.*;
+import javax.swing.border.TitledBorder;
+import java.awt.*;
+import javax.swing.JLabel;
+
+// Assignment 2
+// Class: CS 410
+// Author: Garth Scheck
+// Rev: A.01
+// Date: 5/24/2019
+
+// Sensor Panel Class
+// Displays Sensor Sub Panel
+public class SensorPanel extends JPanel{
+
+    // Constructor, accepts panel label and sensor measurement
+    public SensorPanel(SensorInterface sensor)
+    {
+        double meas = sensor.readValue();

```

Comment:
It would be nicer if there was comments for what the variable name means since "meas" is not easy to understand what the abbreviation indicates. Otherwise, I think you could change the variable name to make it more understandable.

Reply:
I agree.